### DISCUSSION TITLE
is there a way to make a modal to update a record

### DISCUSSION THREAD

**Discussion by docentemaurocasadei** [Laravel-Backpack/community-forum#391] [Q&A (Help)]
Hi, my best goal is to have a modal for update and insert, so usability can significant increase, is it possibile?

**Comment by soluzione-software** [comment]
someone can answer?

**Comment by tabacitu** [comment]
I agree with you - having the Create & Update forms right there on the list (in a modal or drawer) would be nice. 

There is no way to _easily_ do that in Backpack right now (as of v5). To achieve it, you'd have to:
- [create your own custom buttons](https://backpackforlaravel.com/docs/5.x/crud-buttons#creating-a-custom-button-1) (which open a modal that loads the form)
- [create your own custom Operation](https://backpackforlaravel.com/docs/5.x/crud-operations#creating-a-custom-operation-1) (which responds to the modal AJAX request with the form, and to the modal form submit with errors or success)

It can be done, especially by copy-pasting the existing buttons... and by extending the current Create and Update operations... but it's not straightforward and we don't have a tutorial for it. I expect getting it done for simple forms will be pretty quick, but covering edge cases (modals in modals, tabs, etc) would take a long time.

Hope this points you in the right direction, if you really need it.
Cheers!
